streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -122,7 +124,14 @@ public void commit() {
     }
 
     private void flushAndCheckpointState() {
-        stateMgr.flush();
+        try {
+            stateMgr.flush();
+        } catch (final ProcessorStateException e) {
+            if (e.getCause() instanceof ProducerFencedException) {
```

Comment:
The same argument as for `GlobalStateStore` should apply here -- StandbyTask don't write but only read, hence, no `ProducerFencedException` could ever happen here.

Reply:
Oy. Good call. I've fixed it.